docs/rules/prefer-screen-queries.md

```diff
@@ -2,26 +2,30 @@
 
 ## Rule Details
 
-DOM Testing Library (and other Testing Library frameworks built on top of it) exports a `screen` object which has every query (and a `debug` method). This works better with autocomplete and makes each test a little simpler to write and maintain.
+DOM Testing Library (and other Testing Library frameworks built on top of it) exports a `screen` object which has every query (the `container` and a `debug` method). This works better with autocomplete and makes each test a little simpler to write and maintain.
```

Comment:
Minor spelling mistake, I think ? English is not my native language 😄 

```suggestion
DOM Testing Library (and other Testing Library frameworks built on top of it) exports a `screen` object which has every query (plus the `container` and a `debug` method). This works better with autocomplete and makes each test a little simpler to write and maintain.
```

Reply:
It's not mine either. 🙂

Reply:
Agree on @gndelia suggestion as container and debug are extra things apart from the queries, so it's better to say "screen has every query plus container and debug).